Enterprise-CMCS/macpro-mdct-mfp
# [MDCT-2819] Add updated Review and Submit Page

### Description


https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/19439679/bdeb6fbd-28ae-453d-9258-463a75eac2fc

Theres still some snags due to validation so this page won't actually _work_ yet but you can see all the screens and move through them with a little code manipulation. Will be creating tickets for the left over work in JIRA

Tickets created in Jira:
- Update routing and statusing for State and Territory Initiative https://qmacbis.atlassian.net/browse/MDCT-2926
- Ensure submit code submits the form https://qmacbis.atlassian.net/browse/MDCT-2927


### Related ticket(s)


MDCT-2819

---
### How to test

Create a new submission
Navigate to the Review and Submit page
Check it out! 
Hide the disabled field on the submit button to activate the modal to see that
Update line 95 to `{report?.status !== ReportStatus.SUBMITTED ? (` if you want to see the success page

---
### Author checklist
